webapp/components/wpt-prs.js

```diff
@@ -0,0 +1,170 @@
+/**
+ * Copyright 2018 The WPT Dashboard Project. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+
+import '../node_modules/@polymer/iron-collapse/iron-collapse.js';
+import '../node_modules/@polymer/paper-button/paper-button.js';
+import '../node_modules/@polymer/polymer/lib/elements/dom-if.js';
+import '../node_modules/@polymer/polymer/lib/elements/dom-repeat.js';
+import { ensureTrailingSlash } from './utils.js';
+import { html } from '../node_modules/@polymer/polymer/lib/utils/html-tag.js';
+import { PolymerElement } from '../node_modules/@polymer/polymer/polymer-element.js';
+import { LoadingState } from './loading-state.js';
+
+class WPTPR extends PolymerElement {
+  static get template() {
+    return html`
+    <style>
+      .time {
+        font-size: 0.8em;
+        font-style: italic;
+      }
+      .pr {
+        display: flex;
+        align-items: center;
+        margin-bottom: 4px;
+      }
+      .pr img {
+        margin-right: 16px;
+        height: 24px;
+        width: 24px;
+      }
+    </style>
+    <div class="pr">
+      <img src="/static/github.svg">
```

Comment:
You have another icon representing a PR now, right?

Reply:
You probably mean /static/pr_head.svg
Because this list is all PRs, and the link heads to GitHub, I think the GitHub icon is more.. informative?

Reply:
Sure, I didn't think too much about it :)